### ISSUE TITLE
'tuple' object has no attribute 'layer'

### ISSUE THREAD

**Issue by kingdsz** [keras-team/keras#13320]
img_pool = keras.layers.AveragePooling2D(pool_size=(56, 56), strides=(56 ,56), 
    padding='valid')(img_input)
    img_flatten = keras.layers.Flatten()(img_pool)
    se = keras.layers.Dense(2)(img_flatten)
    se = keras.layers.Activation('relu')(se)
    se = keras.layers.Dense(16)(se)
    se = keras.layers.Activation('sigmoid')(se)
    se = keras.layers.Reshape((16,1,1,1))(se)
    x = keras.layers.Reshape((4, 56, 4, 56, 3))(img_input)
    x = keras.layers.Permute((1,3,2,4,5))(x)
    x = keras.layers.Reshape((16,56,56,3))(x)
    x = keras.layers.multiply([x, se])
    x = keras.layers.Reshape((4,4,56, 56, 3))(x)
    x = keras.layers.Permute((1,3,2,4,5))(x)
    x = keras.layers.Reshape(( 224,224,3))(x)

I get an error when I write the above code.'tuple' object has no attribute 'layer'.I don't know what went wrong.

**Comment by amit2555** [comment]
which line of code has the error? Can you paste the traceback here?

**Comment by amit2555** [comment]
this could also be a version mismatch issue so please check that too.
